qa/tasks/cephfs/test_cephfs_shell.py

```diff
@@ -335,6 +335,48 @@ def test_rmdir_p_dir_with_file(self):
         self.mount_a.stat(self.dir_name)
 
 
+class TestRm(TestCephFSShell):
+    dir_name = "test_rm"
```

Comment:
```suggestion
    dir_name = "testdir"
```
Helps understand from logs that argument passed was a directory and not a regular file.